ui/imports/utils/Constants.qml

```diff
@@ -1140,6 +1140,18 @@ QtObject {
         return url.indexOf("DEFAULT-TOKEN") !== -1
     }
 
+    function getSupportedTokenSourceImage(key, useDefault=true) {
+        if (key === "uniswap")
```

Comment:
can we create consts for these somewhere?

Reply:
I'll create this:
```
    readonly property QtObject supportedTokenSources: QtObject {
        readonly property string uniswap: "uniswap"
        readonly property string status: "status"
        readonly property string custom: "custom"
    }
```